pysb/tests/test_bng.py

```diff
@@ -180,6 +180,13 @@ def test_sympy_parameter_keyword():
     generate_equations(model)
 
 
+def test_log():
```

Comment:
The `@with_model` annotation got lost here, probably as part of a conflict resolution. I know because I just did the same thing on a local rebase that added a new test at the end. :)

Reply:
This looks like the only reason CI is failing on this.